src/main/java/io/quarkiverse/spec/generator/deployment/codegen/SpecApiCodeGenUtils.java

```diff
@@ -11,19 +11,24 @@ private SpecApiCodeGenUtils() {
 
     private static final String BUILD_TIME_SPEC_PREFIX_FORMAT = "quarkus.%s.spec.%s";
     private static final String BASE_PACKAGE_PROP_FORMAT = "%s.base-package";
+    private static final String IGNORED_FILES_PROP_FORMAT = "quarkus.%s.codegen.ignore";
 
-    public static String getSanitizedFileName(final Path openApiFilePath) {
+    public static String getSanitizedFileName(final Path specFilePath) {
         return StringUtil
                 .replaceNonAlphanumericByUnderscores(
-                        SpecApiGeneratorOutputPaths.getRelativePath(openApiFilePath).toString());
+                        SpecApiGeneratorOutputPaths.getRelativePath(specFilePath).toString());
     }
 
-    public static String getBuildTimeSpecPropertyPrefix(final Path openApiFilePath, String configPrefix) {
-        return String.format(BUILD_TIME_SPEC_PREFIX_FORMAT, configPrefix, getSanitizedFileName(openApiFilePath));
+    public static String getBuildTimeSpecPropertyPrefix(final Path specFilePath, String configPrefix) {
+        return String.format(BUILD_TIME_SPEC_PREFIX_FORMAT, configPrefix, getSanitizedFileName(specFilePath));
+    }
+    
+    public static String getIgnoredFilesPropFormat (String configPrefix) {
+    	return String.format(IGNORED_FILES_PROP_FORMAT, configPrefix);
```

Comment:
We need to propagate the changes from this PR here: https://github.com/quarkiverse/quarkus-openapi-generator/pull/179

Reply:
In general those changes collide with these ones, even if we remove property reading from here and introduce a predicate, there will be a collision. 
Now, the real question, the point of this library is to reuse logic & property for input processing for both extensions, since we want them to be aligned (specially on regard of what we called stream processor on OpenApi, that is the capability of the customer to provide the inputstreams to be processed) and do not not have a disjointed set of properties

Reply:
That PR has appeal, IMO. And the `include`/`exclude` filter makes sense. Making them implement their streaming processor is a workaround, which I propose, but not a friendly solution.

Reply:
yes, I agree this PR is fine. Actually, if implemented in the library will also be available to AsyncAPI